src/mongocxx/options/insert.hpp

```diff
@@ -50,8 +65,18 @@ class MONGOCXX_API insert {
     ///
     const bsoncxx::stdx::optional<class write_concern>& write_concern() const;
 
+    ///
+    /// The current ordered value for this operation.
+    ///
+    /// @return The current ordered value.
+    ///
+    /// @see http://docs.mongodb.org/manual/reference/method/db.collection.insert/
+    ///
+    const bool ordered() const;
+
    private:
     bsoncxx::stdx::optional<class write_concern> _write_concern;
+    bool _ordered;
```

Comment:
I think we should make this an optional and put the defaulting logic at a higher level, for example on line 627 of collection.hpp.